renovate.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for bigpanda rs repository",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs"
+  ],
+  "schedule": [
+    "before 3am every weekday"
+  ],
+  "enabledManagers": [
+    "github-actions"
+  ],
+  "cargo": {
+    "packageRules": [
+      {
+        "description": "Wait 3 days before opening a PR for new major version updates",
+        "matchUpdateTypes": ["major"]
+      },
+      {
+        "description": "Automerge non-major version updates (assuming the CI pipelines pass)",
+        "matchUpdateTypes": ["patch", "minor"]
+      }
+    ]
+  },
```

Comment:
Looking at the [cargo docs](https://sourcegraph.com/github.com/renovatebot/renovate/-/blob/docs/usage/rust.md) I'm not seeing where the syntax for this `cargo` section is being referenced -- could we add any rust/cargo-specific package logic into the existing `packageRules` block on L27 instead?

Reply:
yes sure added. There is one `Cargo.toml` file in the repo, although it is not inside any package currently. But for future if packages are created then `cargo.packageRules` will help.